### PR TITLE
WIP: Add ValidatePlaceLimitOrder and validateCancelLimitOrder

### PR DESCRIPTION
**Pull request by fdgod09**
## Why this should be merged
Implemented the following in the juror precompile.
```
juror.ValidatePlaceLimitOrderV2
juror.ValidateCancelLimitOrderV2
```

### CODE REVIEW

File: chain.json [hunk 1]
```diff
@@ -8,5 +8,6 @@
   "priority-regossip-addresses": ["0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266", "0x70997970C51812dc3A010C7d01b50e0d17dc79C8", "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC", "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"],
   "validator-private-key-file": "/tmp/validator.pk",
   "is-validator": true,
-  "trading-api-enabled": true
+  "trading-api-enabled": true,
+  "testing-api-enabled": true
```
Comment (fdgod09) [inline chain.json:12]: revert this change

**Review by atvanguard — APPROVED** [review]
